apps/admin/frontend/src/screens/tally/confirm_remove_cvrs_modal.tsx

```diff
@@ -45,18 +44,15 @@ export function ConfirmRemoveCvrsModal({
     onClose();
   }
 
-  if (isTestMode && hasManualData) {
+  if (hasManualData) {
```

Comment:
Currently we have branching logic for test mode, which we want to remove. Also, previously this might have been a bug because the extra button to remove CVRs Only vs. CVRs + Manual Data appears to be conditioned on test mode?

Reply:
I introduced this logic specifically in response to a concern that users might forget to clear manual tallies after finishing L&A testing.

Reply:
handled by discussion and later commit